webpack.common.config.ts

```diff
@@ -143,6 +149,15 @@ module.exports = {
   * once angularjs is removed from corresponding pages.
   */
   plugins: [
+    // This plugin performs a direct text replacement, so the value given to it
+    // must include the surrounding quotes. This is done using JSON.stringify.
+    // See https://webpack.js.org/plugins/define-plugin/
+    new webpack.DefinePlugin({
+      GA_ANALYTICS_ID: JSON.stringify(GA_ANALYTICS_ID),
+      UA_ANALYTICS_ID: JSON.stringify(UA_ANALYTICS_ID),
+      CAN_SEND_ANALYTICS_EVENTS: JSON.stringify(CAN_SEND_ANALYTICS_EVENTS),
```

Comment:
Can you show proof that, after this is run, this results in something that is falsy (i.e. `false`) rather than something like `'false'` which would evaluate to true?

Reply:
This was incorrect, it didn't work correctly when I set it to false, I have removed JSON.stringify for the boolean variable now. Thanks for catching it!